iscript/src/iscript/constants.py

```diff
@@ -22,6 +22,9 @@
         "zipfile_cmd": "ditto",
         "create_pkg": False,
         "hardened_runtime_only_files": ["wg", "wireguard-go"],
+        "profiles": {
```

Comment:
This doesn't appear to be used anywhere? (And if it is, it could use a better name. Perhaps `provisioning_profiles` if that's what they are.)